### PR TITLE
Feat/image text teaser

### PR DESCRIPTION
**Pull request by Wandkleister**

### CODE REVIEW

File: src/_includes/js/image-text-teaser.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+		this.defaultContent = this.container.querySelector('[data-js-item="image-text-teaser-content"]');
+		this.hiddenContent = this.container.querySelector('[data-js-item="image-text-teaser-content-hidden"]');
+		this.forwardBtn = this.defaultContent.querySelector('[data-js-atom="image-text-teaser-forward"]');
+		this.backwardBtn = this.hiddenContent.querySelector('[data-js-atom="image-text-teaser-backward"]');
+
+		this.bindEvents();
+	}
+
+	bindEvents() {
+		console.log(this.defaultContent, this.hiddenContent, this.forwardBtn, this.backwardBtn);
```
Comment (timbru31) [inline src/_includes/js/image-text-teaser.js:16]: möp ;)

File: src/_includes/js/image-text-teaser.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+	moveToHiddenContent() {
+		this.hiddenContent.classList.add("is-active");
+		this.defaultContent.classList.add("is-hidden");
+	}
+
+	moveToDefaultContent() {
+		this.hiddenContent.classList.remove("is-active");
+		this.defaultContent.classList.remove("is-hidden");
+	}
```
Comment (timbru31) [inline src/_includes/js/image-text-teaser.js:29]: If feasible, can those two methods be combined and just use `classList.toggle()` instead?

File: src/_includes/js/image-text-teaser.js (hunk 1 above)
Comment (Wandkleister) [reply src/_includes/js/image-text-teaser.js]: Removed ;)

File: src/_includes/js/image-text-teaser.js (hunk 2 above)
Comment (Wandkleister) [reply src/_includes/js/image-text-teaser.js]: Adjusted to have just one toggle function

**Review by tam-kis — APPROVED** [review]

**Pull request merged**